_docs/faqs.md

```diff
@@ -1,66 +1,59 @@
 ---
-title: FAQs
+title: "Frequently Asked Questions (FAQs)"
 permalink: /docs/faq/
 ---
 
-Frequently asked questions:
+<!-- hopefully the metadata update in "..." works -->
```

Comment:
@socadk, works in general, but leads to linebreak in menu and page:

![image](https://user-images.githubusercontent.com/6763278/77546937-d4b0ac80-6eac-11ea-8f5c-d0b8dfb2d99f.png)

Reply:
Okay for me, but we have to remove this comment.